<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LobbyWidget</class>
 <widget class="QWidget" name="LobbyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="buttonStart">
   <property name="geometry">
    <rect>
     <x>564</x>
     <y>480</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>721</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="horizontalSpacing">
     <number>50</number>
    </property>
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="labelPlayer">
      <property name="text">
       <string>Players:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="textLog">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="listPlayers">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelLog">
      <property name="text">
       <string>Log:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="buttonBack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Powr√≥t</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>721</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layoutHeader">
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="styleSheet">
       <string notr="true">font-size: 30px;
color: white;</string>
      </property>
      <property name="text">
       <string>Lobby</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
